### ISSUE TITLE
[2023/03/15] Is the Performance of My Deep Network Too Good to Be True? A Direct Approach to Estimating the Bayes Error in Binary Classification

### ISSUE THREAD

**Issue by Minosota** [zaemyung/Seeking-SOTA#82]
* 제목: Is the Performance of My Deep Network Too Good to Be True? A Direct Approach to Estimating the Bayes Error in Binary Classification
* 출판정보
   - Takashi Ishida et al., ICLR 2023
* Github: https://github.com/takashiishida/irreducible
* Abstract
> There is a fundamental limitation in the prediction performance that a machine learning model can achieve due to the inevitable uncertainty of the prediction target. In classification problems, this can be characterized by the Bayes error, which is the best achievable error with any classifier. The Bayes error can be used as a criterion to evaluate classifiers with state-of-the-art performance and can be used to detect test set overfitting. We propose a simple and direct Bayes error estimator, where we just take the mean of the labels that show uncertainty of the class assignments. Our flexible approach enables us to perform Bayes error estimation even for weakly supervised data. In contrast to others, our method is model-free and even instance free. Moreover, it has no hyperparameters and gives a more accurate estimate of the Bayes error than several baselines empirically. Experiments using our method suggest that recently proposed deep networks such as the Vision Transformer may have reached, or is about to reach, the Bayes error for benchmark datasets. Finally, we discuss how we can study the inherent difficulty of the acceptance/rejection decision for scientific articles, by estimating the Bayes error of the ICLR papers from 2017 to 2023.

**Comment by Minosota** [comment]
[ML_study_ Estimating the Bayes Error in Binary Classification .pdf](https://github.com/zaemyung/Seeking-SOTA/files/10981972/ML_study_.Estimating.the.Bayes.Error.in.Binary.Classification.pdf)
